src/core/Track.cpp

```diff
@@ -717,19 +717,38 @@ void TrackContentObjectView::mousePressEvent( QMouseEvent * me )
 			}
 			else
 			{
-				m_action = ToggleSelected;
+				gui->songEditor()->m_editor->selectAllTcos( false );
+				QVector<TrackContentObjectView *> tcoViews;
+				tcoViews.push_back( this );
+				DataFile dataFile = createTCODataFiles( tcoViews );
+				QPixmap thumbnail = QPixmap::grabWidget( this ).scaled(
+							128, 128,
+							Qt::KeepAspectRatio,
+							Qt::SmoothTransformation );
+				new StringPairDrag( QString( "tco_%1" ).arg(
+										m_tco->getTrack()->type() ),
+									dataFile.toString(), thumbnail, this );
 			}
```

Comment:
okay, but now you can't select/deselect a single TCO by [CTRL] + Click. That's why I moved this whole block into the mouseMoveEvent(). There we can test if the mouse moved 2 pixels since click(copy TCO) or not(toggleSelected). See: https://github.com/LMMS/lmms/pull/3988/files

Reply:
Thanks! I'll look into that.

Reply:
Done via 8434d5f.